.circleci/config.yml

```diff
@@ -74,36 +75,19 @@ jobs:
 
   e2e-tests:
     <<: *defaults
+    docker:
+      - image: cypress/included:12.13.0
     steps:
       - checkout
-      - restore-cache: *restore-yarn-cache
+      - restore_cache: *restore-yarn-cache
       - run: *yarn
-      - save-cache: *save-yarn-cache
+      - save_cache: *save-yarn-cache
       - run:
           name: E2E tests
           command: yarn test:e2e
 
```

Comment:
the smallest of small things: there's an extra line break between these sections